src/field/field3d.cxx

```diff
@@ -143,8 +145,8 @@ void Field3D::splitYupYdown() {
   }
 
   for (int i = 0; i < fieldmesh->ystart; ++i) {
-    yup_fields.emplace_back(fieldmesh);
-    ydown_fields.emplace_back(fieldmesh);
+    yup_fields.emplace_back(emptyFrom(*this));
+    ydown_fields.emplace_back(emptyFrom(*this));
```

Comment:
Note that this now immediately allocates. I'm not sure we want to do this here as it should be done in the parallel transforms.

Although maybe it doesn't hurt as it shouldn't actually allocate memory in most cases.

Reply:
Still, it wouldn't hurt to just use the full constructor here to avoid the `allocate()`. It might save a bit of memory usage in some cases. I'll update.

Reply:
I'm not even sure we need the full constructor -- just some kind of object in the vector that we can assign to. The transforms don't (or shouldn't?) update the field, but clobber it completely.